communication/youtube/youtube-guidelines.md

```diff
@@ -56,191 +40,222 @@ Hours, will be shared with the appropriate point(s) of contact.
 
 ### Uploading Guidelines for Collaborators
 
-**NOTE:** If you're using a G Suite account you may need to [loosen the
-permissions in your YouTube settings]. If you have any questions reach out to
-the [YouTube admins] or [SIG Contributor Experience].
+**NOTE:** If you're using a Google Workspace account (formerly known as G Suite)
+you may need to [update the permissions in your YouTube settings][]. If you have
+any questions, reach out to the [YouTube admins][] or
+[SIG Contributor Experience][]. You may need to reach out to someone at your
+organization if you do not have access to Google Workspace Admin permissions.
 
-**NOTE:** Both public and private steering meeting recordings should be made public.
+**NOTE:** Both public and private steering meeting recordings should be made
+public.
 
 With collaboration comes great responsibility. Playlist collaborators in the
 community must use it responsibly and are subject to the following guidelines:
 
-- Group leaders or other appropriate point(s) of contact are the primary
-  managers for the playlist, once collaboration is configured. YouTube admins
-  should **only** be contacted if the issue cannot be resolved by one of the
-  playlist owners.
+-   Group leaders or other appropriate point(s) of contact are the primary
+    managers for the playlist, once collaboration is configured. YouTube admins
+    should **only** be contacted if the issue cannot be resolved by one of the
+    playlist owners.
+
+-   Upload responsibilities belong to the group leaders or other appropriate
+    contacts. YouTube admins should **only** be contacted if the issue cannot be
+    resolved by the playlist owners.
+
+-   Please post only related content, for example meeting recordings, in the
+    appropriate playlists.
+    -   Posting of any exceedingly inappropriate content (i.e. NSFW content)
```

Comment:
I think the word exceedingly should be removed. This sounds as though if it is not exceedingly, it is appropriate.